ReactWindows/ReactNative/Bridge/DispatcherHelpers.cs

```diff
@@ -20,16 +20,16 @@ public static bool IsOnDispatcher()
             return CoreWindow.GetForCurrentThread()?.Dispatcher != null;
         }
 
-        public static async void RunOnDispatcher(DispatchedHandler action)
+        public static async Task RunOnDispatcherAsync(DispatchedHandler action)
```

Comment:
>Task [](start = 28, length = 4)

Why does this need to return Task?  This is intended to be fire and forget to queue up a task on the dispatcher.  If they want to await the response, that's what CallOnDispatcher is for (now CallOnDispatcherAsync, which is fine)